inspire/utils/bibtex_booktitle.py

```diff
@@ -43,8 +43,8 @@ def generate_booktitle(record):
                         if acronym:
                             booktitle = "%s: %s" % (rn, acronym, )
                         else:
-                            recids = search_pattern(p="reportnumber:%s"
-                                                    % (rn,))
+                            recids = Query(p="reportnumber:%s" % (rn,))\
```

Comment:
I think if you want the same behaviour as before this should be:

```python
Query(p="reportnumber:%s" % (rn,)).search().recids
```

In order to get record ids back. Otherwise you might receive back the full json by default. Did you check?